package.json

```diff
@@ -4,7 +4,7 @@
   "description": "An open source eCommerce platform powered by the MERN sttack. It is designed to allow anyone to host their own online shop with ease.",
   "main": "server.js",
   "scripts": {
-    "format": "prettier {,*/**/}*.{js,jsx,ts,tsx,json,html} --write",
+    "format": "prettier --write \"{,*/**/,*}*.{js,jsx,ts,tsx,json,html}\"",
```

Comment:
Thanks for working through this with me last night to make sure this works not only on Linux / WSL but MacOS also!